### PR TITLE
Add support for aes

### PR DESCRIPTION
**Pull request by plcstpierre**
Add wrapper for aes functions.

### CODE REVIEW

**Review by martelogan — APPROVED** [review]
Very cool 👌

File: pkg/crypto/aes/aes.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	l.Pop(1)
+}
+
+var aesLibrary = []lua.RegistryFunction{
+	{"encryptCBC", encryptCBC},
+	{"decryptCBC", decryptCBC},
+}
+
+func encryptCBC(l *lua.State) int {
+	key, _ := hex.DecodeString(lua.CheckString(l, 1))
```
Comment (etiennemartin) [inline pkg/crypto/aes/aes.go:28]: Do we care to report any of the errors along the way? We might want to consider checking and returning 0 for failures along the way. Maybe even add some logging?

File: pkg/crypto/aes/aes.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	encrypter.CryptBlocks(ciphertext[aes.BlockSize:], content)
+
+	l.PushString(hex.EncodeToString(ciphertext))
+
+	return 1
+}
+
+func decryptCBC(l *lua.State) int {
+	key, _ := hex.DecodeString(lua.CheckString(l, 1))
+	ciphertext, _ := hex.DecodeString(lua.CheckString(l, 2))
```
Comment (etiennemartin) [inline pkg/crypto/aes/aes.go:56]: Same comment here around error reporting.

File: pkg/crypto/aes/aes.go [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+	block, err := aes.NewCipher(key)
+	if err != nil {
```
Comment (etiennemartin) [inline pkg/crypto/aes/aes.go:59]: This is a great example of how we could report errors in other places where errors are dropped 👌🏻

**Review by etiennemartin — COMMENTED** [review]
Overall the code looks great! I would add a bit more error handling to ensure that we are able to track issues if something goes wrong.

**Pull request merged**